exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/conv/DummyConvertTo.java

```diff
@@ -32,6 +34,8 @@
 @FunctionTemplate(name = "convert_to", scope = FunctionScope.SIMPLE, nulls = NullHandling.NULL_IF_NULL)
 public class DummyConvertTo implements DrillSimpleFunc {
 
+  @Param VarCharHolder in1;
```

Comment:
Initially, I needed to add "@Param". Otherwise, validation will fail due to failure of finding a function with the correct number of argument.

Now, with this 
https://github.com/hsuanyi/incubator-drill/blob/DRILL-4372_For_Review/exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/DrillSqlOperator.java#L103
we can just remove this line.